include/deal.II/fe/mapping_fe_field.h

```diff
@@ -115,54 +115,65 @@ class MappingFEField : public Mapping<dim,spacedim>
    */
   MappingFEField (const MappingFEField<dim,spacedim,VECTOR,DH> &mapping);
 
+  /**
+   * Return a pointer to a copy of the present object. The caller of this copy
+   * then assumes ownership of it.
+   */
+  virtual
+  Mapping<dim,spacedim> *clone () const;
+
+
 
   /**
-   * Transforms the point @p p on the unit cell to the point @p p_real on the
-   * real cell @p cell and returns @p p_real.
+   * Always returns @p false.
    */
-  virtual Point<spacedim>
-  transform_unit_to_real_cell (
-    const typename Triangulation<dim,spacedim>::cell_iterator &cell,
-    const Point<dim>                                 &p) const;
+  virtual
+  bool preserves_vertex_locations () const;
+
 
   /**
-   * Transforms the point @p p on the real cell to the point @p p_unit on the
-   * unit cell @p cell and returns @p p_unit.
-   *
-   * Uses Newton iteration and the @p transform_unit_to_real_cell function.
-   *
-   * In the codimension one case, this function returns the normal projection
-   * of the real point @p p on the curve or surface identified by the @p cell.
+   * @name Mapping points between reference and real cells
+   * @{
+   */
+
+  // for documentation, see the Mapping base class
```

Comment:
If doxygen sees no doxygen-style comment in front of a virtual function, it copies the comment from the base class. This is exactly what I want -- using a doxygen-style comment here would defeat the purpose.

Of course, I could have copied the comment from the base class myself, but then I would have to keep the original and multiple copies in synch. Or I could have had no comment (of any style) here at all, but the perfectionist in me does not allow for that :-)